### DISCUSSION TITLE
HX711 SIC or DAT not available

### DISCUSSION THREAD

**Discussion by BruceParmele** [arendst/Tasmota#20766] [Support request] [answered]
I set up a keg monitoring system using load cells and an HX711 amp. It worked fine until I shorted it out. So I built a new system with the new components. But now I am unable to locate the HX711 options for under GENERIC(18) or any other module for that matter. I am lost. Any help is most appreciated

**Comment by sfromis** [comment, marked as answer]
For HX711 support, you need to use a build enabling the driver. That could be the precompiled tasmota-sensors variant.

**Reply by BruceParmele** [reply]
Thank you!  I just realized I was using the wrong bin.  I am a happy camper now.  I flashed with the tasmota-sensor.bin and I am back in business.
